src/lib/middleware/intercom.ts

```diff
@@ -2,7 +2,7 @@ import * as crypto from "crypto"
 // @ts-ignore
```

Comment:
Not for this PR, but... do we use intercom anymore? I thought it was removed. 🤔 

@joeyAghion?

Reply:
It was abandoned but not removed (tracked in [PURCHASE-2222](https://artsyproduct.atlassian.net/browse/PURCHASE-2222)).